cloud/microsoft/office365/teams/mode/devicesusage.pm

```diff
@@ -58,7 +62,7 @@ sub custom_active_threshold {
 sub custom_active_output {
     my ($self, %options) = @_;
 
-    my $msg = sprintf("Active users on %s : %d/%d (%.2f%%)",
+    my $msg = sprintf("Active devices on %s : %d/%d (%.2f%%)",
```

Comment:
Same as for Skype.

Reply:
See [above](https://github.com/centreon/centreon-plugins/pull/2042#discussion_r464733505).